### PR TITLE
Add per-column sort order.

### PR DESCRIPTION
**Pull request by omp**
This allows you to change the default sort order of a particular column using the `data-sort-order` attribute (`asc` or `desc`).

### CODE REVIEW

**Comment by tristen** [comment]
:+1: Can you document this change in the README?

**Comment by omp** [comment]
Sure, just added a description and example to the ascending/descending section of the README. Hopefully it's good enough. :)

File: src/tablesort.js [hunk 1]
```diff
@@ -140,6 +141,10 @@
           sortDir = 'sort-down';
         } else if (header.classList.contains('sort-down')) {
```
Comment (xPaw) [inline src/tablesort.js]: Perhaps instead of relying on the class for sort ordering it should solely use the data attribute?

**Comment by xPaw** [comment]
:+1:

**Pull request merged**